presto-main/src/test/java/io/prestosql/metadata/AbstractMockMetadata.java

```diff
@@ -686,4 +686,10 @@ public AnalyzePropertyManager getAnalyzePropertyManager()
     {
         return Optional.empty();
     }
+
+    @Override
+    public boolean supportsMissingColumnsOnInsert(Session session, TableHandle tableHandle)
```

Comment:
here & in the interface -- move right after `beginInsert(Session session, TableHandle tableHandle, List<ColumnHandle> columns)`